scripts/ci_install_deps

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
-curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+curl -sSL https://install.python-poetry.org | python
 . $HOME/.poetry/env
 poetry --version
 poetry config virtualenvs.in-project true
 poetry run python -m ensurepip --upgrade
+poetry update
```

Comment:
```suggestion
```

This would always update all dependencies in CI meaning that local testing would not match what CI is running—which can lead to confusion when there are test failures in one place and not another. Instead, I should set up Renovate or similar for this repo to keep things rolling 😁

Reply:
OK sure, I'll revert the change and let you push to the branch instead?

Reply:
Yeah, did you need the newer qiskit or just wanted to test on it? I can get Renovate set up and have it update for us instead of doing it by hand if you don't need the newer version for this branch.

Reply:
Yeah I need the new version, the qiskit API has changed quite a bit since 0.27.

Reply:
I opened #31 which unbinds the upper dependency of qiskit since we don't have anything in place to auto-bump it yet. Not sure how long it'll be until we can hook up Renovate.

Although, I also notice that we have several other open conflicting PRs which bump the minimum version to 0.34, so that'll require some reconciliation 😅.